### PR TITLE
remove python-* in base.yaml, which already defined in python.yaml

### PR DESCRIPTION
**Pull request by k-okada**
rewrited version of https://github.com/ros/rosdistro/pull/15257

some of rosdep keys is multiply defined, and that sometime confuse users.

- This just to remove duplicates entry from base.yaml, no regressions.

### CODE REVIEW

**Review by tfoote — APPROVED** [review]

**Comment by ipa-mdl** [comment]
To avoid further confusion, what about moving all `python-*` keys into python.yaml by policy?

File: rosdep/base.yaml [hunk 1]
```diff
@@ -3568,48 +3568,14 @@ pyqt5-dev-tools:
...
-  opensuse: [python-pygraphviz]
 python-qrencode:
   debian: [python-qrencode]
   gentoo: [media-gfx/qrencode-python]
   ubuntu: [python-qrencode]
-python-sphinx:
-  fedora: [python-sphinx]
-  gentoo: [dev-python/sphinx]
-python-tornado:
-  arch: [python-tornado]
```
Comment (mikaelarguedas) [inline rosdep/base.yaml]: Please add the arch key to the python.yaml file

**Review by mikaelarguedas — CHANGES_REQUESTED** [review]
One rule is removed here and doesn't exist in the python.yaml file.

File: rosdep/base.yaml (hunk 1 above)
Comment (k-okada) [inline rosdep/base.yaml]: thanks for catch! done

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks for iterating

**Pull request merged**